api/institutions/serializers.py

```diff
@@ -12,6 +12,7 @@ class InstitutionSerializer(JSONAPISerializer):
     name = ser.CharField(required=False)
     id = ser.CharField(required=False, source='_id')
     logo_path = ser.CharField()
+    auth_url = ser.CharField()
```

Comment:
Is this field required? If so, explicitly pass `required=True`. Also it wouldn't hurt to pass in `help`.